backend/app/api/v1/commons/ocp.py

```diff
@@ -27,7 +27,7 @@ async def getData(start_datetime: date, end_datetime: date, size:int, offset:int
     tasks = [item['_source'] for item in response["data"]]
     jobs = pd.json_normalize(tasks)
     if len(jobs) == 0:
-        return jobs
+        return ({'data':jbs, 'total': 0})
```

Comment:
I think that `'data':jbs` is a typo. There isn't actually a `jbs` in scope, is there?